main.nf

```diff
@@ -51,13 +51,17 @@ WorkflowMain.initialise(workflow, params, log)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-include { READSIMULATOR } from './workflows/readsimulator'
+include { READSIMULATOR } from './workflows/readsimulator/main'
 
 //
 // WORKFLOW: Run main nf-core/readsimulator analysis pipeline
 //
 workflow NFCORE_READSIMULATOR {
-    READSIMULATOR ()
+
+    ch_input    = Channel.fromSamplesheet("input")
+    ch_input.view()
```

Comment:
Does `ch_input.view()` need to be removed? I assume this was added while testing because I often use .view() when I'm testing Nextflow pipelines.

Reply:
Good catch!